website/content/api-docs/catalog.mdx

```diff
@@ -59,17 +59,17 @@ The table below shows this endpoint's support for
   provided, it will be defaulted to the value of the `Service.Service` property.
   Only one service with a given `ID` may be present per node. We recommend using
   [valid DNS labels](https://en.wikipedia.org/wiki/Hostname#Restrictions_on_valid_hostnames)
-  for service definition names for [compatibility with external DNS](/docs/discovery/services#service-and-tag-names-with-dns).
+  for service definition names for [compatibility with external DNS](/consul/docs/discovery/services#service-and-tag-names-with-dns).
   The service `Tags`, `Address`, `Meta`, and `Port` fields are all optional. For more
   information about these fields and the implications of setting them,
-  see the [Service - Agent API](/api-docs/agent/service) page
+  see the [Service - Agent API](/consul/api-docs/agent/service) page
   as registering services differs between using this or the Services Agent endpoint.
 
 - `Check` `(Check: nil)` - Specifies to register a check. The register API
   manipulates the health check entry in the Catalog, but it does not setup the
   script, TTL, or HTTP check to monitor the node's health. To truly enable a new
   health check, the check must either be provided in agent configuration or set
-  via the [agent endpoint](agent).
+  via the [agent endpoint](https://consul.io/agent).
```

Comment:
🎏  Need to look into this one

Reply:
Looks like it should be relative

Reply:
Looks like on prod its the same behavior, a broken link. Will note as something to fix as a follow up

Reply:
This is an instance of a "no-dots" link. The way those are handled, is that it looks for the URL (`agent` in this case) within the page path (`/api-docs/cataglog` in this case). So here, because `agent` is not in the page/file path, the link is getting prefixed with the .io hostname. This can be categorized as a content bug and since the bug is present in prod as well, it can be fixed in a PR before or after this one.